.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
```

Comment:
Let's add a new line after this, we've always put the different repos on their own.